docs/configuration/model/ModelConfiguration.md

```diff
@@ -186,3 +186,50 @@ The docs button which is displayed in the scenario properties modal window is by
 ``` scenarioPropertiesDocsUrl: "http://custom-configurable-link"```
 
 in the config. 
+
+## Multitenancy support
+
+Nussknacker supports multitenancy, allowing multiple Nussknacker designer instances to operate on shared infrastructure components,
+such as Kafka, Flink, or InfluxDB. This is achieved by using configured namespaces to isolate resources. For example:
+- Kafka Topics (`kafkaTopic`): Prefixed with a namespace.
+- Kafka Consumer Groups (`kafkaConsumerGroup`): Prefixed with a namespace
+- Flink Jobs (`flink`): Prefixed with a namespace.
+- InfluxDB Metrics (`metrics`): Tagged with an additional namespace tag.
```

Comment:
Looking at code, I think this works only for Flink, not for Lite?

Reply:
Done.